.eslintrc.json

```diff
@@ -7,7 +7,14 @@
     "jest": true,
     "node": true
   },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["node", "standard", "prettier"],
+  "extends": [
+    "eslint:recommended",
+    "standard",
+    "prettier",
+    "prettier/standard",
+    "plugin:prettier/recommended"
```

Comment:
`"extends": ["plugin:prettier/recommended"]` already takes care of loading the `prettier` eslint plugin, so it should not be necessary to specify it again in the `plugins` array, see: https://prettier.io/docs/en/integrating-with-linters.html#recommended-configuration